<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询日程</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/ico/date.ico</normaloff>:/ico/date.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbb_method">
       <item>
        <property name="text">
         <string>-请选择输入方式-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主题</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>地点</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>备注</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cnt"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
